packages/beacon-node/test/perf/eth1/pickEth1Vote.test.ts

```diff
@@ -4,95 +4,97 @@ import {ContainerType, ListCompositeType} from "@chainsafe/ssz";
 import {newFilledArray, BeaconStateAllForks} from "@lodestar/state-transition";
 import {fastSerializeEth1Data, pickEth1Vote} from "../../../src/eth1/utils/eth1Vote.js";
 
-describe("eth1 / pickEth1Vote", () => {
-  const ETH1_FOLLOW_DISTANCE_MAINNET = 2048;
-  const EPOCHS_PER_ETH1_VOTING_PERIOD_MAINNET = 64;
-  const SLOTS_PER_EPOCH_MAINNET = 32;
-  const eth1DataVotesLimit = EPOCHS_PER_ETH1_VOTING_PERIOD_MAINNET * SLOTS_PER_EPOCH_MAINNET;
+describe("pickEth1Vote", () => {
+  describe("eth1 / pickEth1Vote", () => {
```

Comment:
Please do not nest unless it's really necessary

Reply:
This is added because of the lint error from the recommended linting rules. See the above comment for details.